### DISCUSSION TITLE
The selected project is already linked to another Cluster. ERROR

### DISCUSSION THREAD

**Discussion by neyneyalldayday** [vercel/community#2291] [Help] [answered]
### Summary

im trying to integrate my database with vercel. i keep getting the error "The selected project is already linked to another Cluster." even when its a brand new project and i only have one cluster in mongo. anyone run into this issue?

### Additional information

it only happens when trying to integrate in vercel.

**Comment by amyegan** [comment, marked as answer]
Hi @neyneyalldayday! It sounds like you already have an integration that you need to remove before you can link it to your new cluster. 

MongoDB has some documentation on [how to disconnect an integration](https://www.mongodb.com/docs/atlas/reference/partner-integrations/vercel/#std-label-disconnect-vercel). If you try those steps and still have trouble, it may be best to reach out to MongoDB for assistance as the integration is maintained by them and not by Vercel.

**Comment by neyneyalldayday** [comment]
Yeah I think the issue was I put the Mongod_uri env in before trying to
integrate so it was throwing the error. It's all sorted now. BUT I still
have no data in my deployment. So troubleshooting that now.

**Reply by icjosh10** [reply]
I cannot believe that was the issue... spent hours on this... smh

**Reply by binnehaha321** [reply]
you save my life @neyneyalldayday

**Reply by blazejmojak** [reply]
thanks @neyneyalldayday - I appreciate you wrote about that and no left that for yourself. Saved me hours.

**Reply by rahulamlekar** [reply]
If you just migrated your project and don't see any environment variables transferred over, wait for an hour or so. It's still in their system but you just can't see it yet.

It popped up for me after an hour and then the above comment finally made sense.
